test/e2e/suite_test.go

```diff
@@ -138,101 +115,101 @@ var _ = ginkgo.BeforeSuite(func() {
 var _ = ginkgo.AfterSuite(func() {
 	// Default storage driver configuration is CSI. Freshly built
 	// CSI driver is installed for that case.
-	if isTestingMigration || isUsingInTreeVolumePlugin {
-		cmLog := testCmd{
-			command:  "bash",
-			args:     []string{"test/utils/controller-manager-log.sh"},
-			startLog: "===================controller-manager log=======",
-			endLog:   "===================================================",
-		}
-		execTestCmd([]testCmd{cmLog})
+	if testconsts.IsTestingMigration || testconsts.IsUsingInTreeVolumePlugin {
+		cmLog := testtypes.TestCmd{
+			Command:  "bash",
+			Args:     []string{"test/utils/controller-manager-log.sh"},
+			StartLog: "===================controller-manager log=======",
+			EndLog:   "===================================================",
+		}
+		testutil.ExecTestCmd([]testtypes.TestCmd{cmLog})
 	}
 
-	if isTestingMigration || !isUsingInTreeVolumePlugin {
-		checkPodsRestart := testCmd{
-			command:  "bash",
-			args:     []string{"test/utils/check_driver_pods_restart.sh", "log"},
-			startLog: "Check driver pods if restarts ...",
-			endLog:   "Check successfully",
+	if testconsts.IsTestingMigration || !testconsts.IsUsingInTreeVolumePlugin {
+		checkPodsRestart := testtypes.TestCmd{
+			Command:  "bash",
+			Args:     []string{"test/utils/check_driver_pods_restart.sh", "log"},
+			StartLog: "Check driver pods if restarts ...",
+			EndLog:   "Check successfully",
 		}
-		execTestCmd([]testCmd{checkPodsRestart})
+		testutil.ExecTestCmd([]testtypes.TestCmd{checkPodsRestart})
 
 		os := "linux"
 		cloud := "azurepubliccloud"
-		if isWindowsCluster {
+		if testconsts.IsWindowsCluster {
 			os = "windows"
 		}
-		if isAzureStackCloud {
+		if testconsts.IsAzureStackCloud {
 			cloud = "azurestackcloud"
 		}
-		createExampleDeployment := testCmd{
-			command:  "bash",
-			args:     []string{"hack/verify-examples.sh", os, cloud},
-			startLog: "create example deployments",
-			endLog:   "example deployments created",
+		createExampleDeployment := testtypes.TestCmd{
```

Comment:
We don't need to create these example resources in the scale test, these are tested in e2e.

Reply:
This is the test/e2e/suite_test.go file? It is required here right?

Reply:
My bad! I thought I was looking at scale_test